packages/ripple-binary-codec/src/types/currency.ts

```diff
@@ -25,11 +25,6 @@ function isIsoCode(iso: string): boolean {
 
 function isoCodeFromHex(code: Buffer): string | null {
   const iso = code.toString()
-  if (iso === 'XRP') {
-    throw new Error(
```

Comment:
This should still throw. If someone tries to encode `XRP` as a 3 letter currency code, it should fail.

Reply:
I think it's fine if it doesn't throw - rippled will catch it